Src/Plugins/Security/SEC1003.ReviewPotentiallySensitiveFiles.json

```diff
@@ -4,16 +4,111 @@
       "Name": "ReviewPotentiallySensitiveFiles",
       "Id": "SEC1003",
       "Level": "Error",
-      "FileNameAllowRegex": "(?i)\\.(pem|pkcs12|p12|pfx|asc|ovpn|rdp|mdf|sdf|sqlite|sqlite3|bek|tpm|fve|jks|psafe3|agilekeychain|kwallet|tblk|ppk)$",
+      "FileNameAllowRegex": "(?i)\\.(agilekeychain|asc|bek|fve|jks|kwallet|mdf|ovpn|p12|pem|pfx|pkcs12|ppk|psafe3|rdp|sdf|sqlite|sqlite3|tblk|tpm)$",
       "Description": "Review potentially sensitive files in versioned engineering content and other locations for security risk.",
       "Message": "'{0:scanTarget}' is {1:fileKind} that may compromise security if visible to attackers.",
-
       "MatchExpressions": [
+        {
+          "SubId": "1PasswordFile",
```

Comment:
>1PasswordFile [](start = 20, length = 13)

Uh-oh! Our validator convention will look for {SubId}Validator as a type name and C# types can't be prefixed with a number. :) So it won't be possible to author a validator for this one currently. We can worry about this later. :)

Reply:
I can change to OnePasswordFile. thats fine.

---
In reply to: [545472151](https://github.com/microsoft/sarif-pattern-matcher/pull/39#discussion_r545472151) [](ancestors = 545472151)